src/app/application/components/route-view/RouteView.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div
+    v-if="!hasParent"
+    class="route-view-title sr-only"
+    aria-live="assertive"
+    aria-atomic="true"
+  >
+    {{ t('components.route-view.route-announcer', {title: title}) }}
+  </div>
+  <slot
+    name="default"
+    :route="{
+      params: Object.fromEntries(Object.entries(route.params).map(([prop, value]) => {
+        return [
+          prop,
+          urlParam(value)
+        ]
+      }))
+    }"
+  />
+</template>
+<script lang="ts" setup>
+import { provide, inject, ref } from 'vue'
+import { useRoute } from 'vue-router'
+
+import { useI18n } from '@/utilities'
+export interface RouteView {
+  addTitle: (title: string, sym: Symbol) => void
+  removeTitle: (sym: Symbol) => void
+}
+const [
+  { t },
+] = [
+  useI18n(),
+]
```

Comment:
**Suggestion**: Can we consider to stop using this pattern of wrapping `use*` functions in an array only to destructure them again? It’s just more code to write and I can’t think of a reason where it would be useful.

```suggestion
const { t } = useI18n()
```

Reply:
I was actually unsure which way to go 😆 tbh I don't care either way at the moment but I might care more later on.

Reasons:

When you get lots of these there's an opinion that says its easier to do like this:

```javascript
const [
  { t },
  service,
  service1,
  ...
] = [
  useI18n(),
  useService(),
  useService1(),
  ...
]
```

which keeps them all together, rather like imports at the top of a file, or arguments in the `()` of a function, and its definitely less code (not that that matters too much)

But as always there's an other opinion that says its not easier, and tbh I'm not totally sure which camp I'm in as of this moment. Happy to change here as I don't mind either way, but I might decide I do mind later.